tests/e2e/checkout.spec.js

```diff
@@ -0,0 +1,25 @@
+context('Checkout Pagarme', () => {
+  describe('when create a purchase with checkout pagar.me as payment method', () => {
+    before(() => {
+      cy.loginAsAdmin()
+      cy.enableCheckoutPagarme()
+      cy.addToCart()
+      cy.goToCheckoutPage()
+      cy.fillCheckoutForm()
+      cy.selectCreditCard()
+      cy.wait(2000)
+      cy.placeOrder()
+
+      cy.fillPagarMeCheckoutCreditCardForm(1)
+    })
+
+    it('should be at order received page', () => {
+      cy.url({ timeout: 60000 }).should('include', '/finalizar-compra/order-received/')
```

Comment:
Será que faz sentido um timeout de 60 segundos aqui? Não é muito? Não sei, estou mais trazendo a questão aqui.

Reply:
O timeout é só pra garantir que a página irá carregar
Isso não influência no desempenho dos testes, e eles falharão de qualquer forma se o elemento não for carregado

Reply:
Ok!